packages/react-instantsearch/src/connectors/connectToggle.js

```diff
@@ -109,14 +109,16 @@ export default createConnector({
     const checked = getCurrentRefinement(props, searchState, this.context);
     const items = [];
     const index = getIndex(this.context);
+
     if (checked) {
       items.push({
```

Comment:
really weird conditional pushing IMO

Reply:
Which behaviour you will expect instead?

Reply:
```js
const items = checked ? [{..stuff}] : []
```

I'm not sure if it's that much better though

Reply:
Yep not sure, because the ternary will be split on several line.